pallets/security/src/default_weights.rs

```diff
@@ -0,0 +1,59 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for security.
+pub trait WeightInfo {
+	fn on_initialize() -> Weight;
+	fn set_parachain_status() -> Weight;
+	fn insert_parachain_error() -> Weight;
+	fn remove_parachain_error() -> Weight;
+	
+}
+
+/// Weights for security using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	fn on_initialize() -> Weight {
+		// Minimum execution time: 1_000 nanoseconds.
+		Weight::from_ref_time(1_000_000_u64)
```

Comment:
Just out of curiosity, how did you come up with this number? Is there some official documentation for this?